cases/src/org/commcare/cases/model/Case.java

```diff
@@ -290,6 +299,8 @@ public void removeIndex(String indexName) {
 
         if (toRemove != null) {
             indices.removeElement(toRemove);
+            return true;
         }
+        return false;
```

Comment:
This line isn't covered by the test suite; pretty trivial line, but might be worth adding in